### PR TITLE
Real closed dev update dependencies

### PR DESCRIPTION
**Pull request by CohenCyril**
Depends on PR #308

### CODE REVIEW

**Comment by thery** [comment]
extra parenthesis in

https://github.com/CohenCyril/opam-coq-archive/blob/97f7f9081488b8cfe939827ff8446b9ed971202b/extra-dev/packages/coq-mathcomp-real-closed/coq-mathcomp-real-closed.dev/opam#L13

File: extra-dev/packages/coq-mathcomp-real-closed/coq-mathcomp-real-closed.dev/opam [hunk 1]
```diff
@@ -10,7 +10,7 @@ license: "CeCILL-B"
 build: [ make "-j" "%{jobs}%" ]
 install: [ make "install" ]
 remove: [ "sh" "-c" "rm -rf '%{lib}%/coq/user-contrib/mathcomp/real_closed'" ]
-depends: [ "coq-mathcomp-field" { = "dev" } ]
+depends: [ "coq-mathcomp-field" { (>= "1.7.0") | (= "dev"))} ]
```
[the remark this answers is not in the document]
Comment (CohenCyril) [reply extra-dev/packages/coq-mathcomp-real-closed/coq-mathcomp-real-closed.dev/opam]: Could you fix it?

[the remark this answers is not in the document]
Comment (CohenCyril) [reply extra-dev/packages/coq-mathcomp-real-closed/coq-mathcomp-real-closed.dev/opam]: Please?

[the remark this answers is not in the document]
Comment (thery) [reply extra-dev/packages/coq-mathcomp-real-closed/coq-mathcomp-real-closed.dev/opam]: sorry I was offline, I take care of it

**Pull request merged**